crates/contract/tests/sandbox/common.rs

```diff
@@ -418,16 +394,17 @@ pub fn new_bls12381() -> (dtos::PublicKey, ckd::KeygenOutput) {
     (pk, keygen_output)
 }
 
-pub async fn init_env_ed25519(
+pub async fn init_env(
     num_domains: usize,
+    scheme: SignatureScheme,
```

Comment:
Can we pass a vector of schemes and create domains accordingly? Or will that break to many tests / blow-up the refactor?